dotCMS/src/main/java/com/dotcms/jobs/business/api/JobQueueManagerAPIImpl.java

```diff
@@ -218,7 +202,11 @@ public void start() {
                 });
             }
 
-            Logger.info(this, "JobQueue has been successfully started.");
+            // Start the abandoned job detector
+            abandonedJobDetector.start();
+
+            Logger.info(this,
```

Comment:
is this going to log every 5 mins? couldn't it be logging too much?

Reply:
No, this is logged only when the `JobQueueManagerAPIImpl` starts, should be just once.